app/page.tsx

```diff
@@ -1,9 +1,14 @@
 import Hero from "@/components/Hero";
+import PartnerBenefits from "@/components/PartnerBenefits";
 
 export default function Home() {
   return (
-    <main className="bg-background px-20">
-      <Hero />
-    </main>
+    <>
+      <main className="bg-background px-20">
+        <Hero />
+      </main>
+
+      <PartnerBenefits />
```

Comment:
Put <PartnerBenefits />  inside main div

Reply:
Can i remove the padding? It doesn't seem to affect the website and its the reason i didn't place my component inside of the main div, since it needs to go from side to side. I also tried to make my component "ignore" the padding but hasn't able to accomplish it.

Reply:
Yes you can do it